### PR TITLE
Feature/pytest to product catalogue

### PR DESCRIPTION
**Pull request by jacoor**

### CODE REVIEW

File: Dshop/Dshop/settings.py [hunk 1]
```diff
@@ -39,6 +39,10 @@
 ]
 
 # Application definition
+PROJECT_APPS = [
+    'apps.core',
+    'apps.users',
+]
```
Comment (RL89pl) [inline Dshop/Dshop/settings.py]:
```suggestion
PROJECT_APPS = [
    'apps.core',
    'apps.users',
    'apps.products_catalogue',
]
```

**Review by RL89pl — CHANGES_REQUESTED** [review]
Updated project_apps

**Comment by jacoor** [comment]
Still, conflicts needs to be fixed.

File: Dshop/Dshop/settings.py (hunk 1 above)
Comment (jacoor) [reply Dshop/Dshop/settings.py]: Conflict fixed manually.

**Pull request merged**